crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -1127,7 +1219,7 @@ mod tests {
     }
 
     #[async_test]
-    async fn get_dm_room_returns_the_room_we_have_with_this_user() {
+    async fn test_get_dm_room_returns_the_room_we_have_with_this_user() {
```

Comment:
I see you are continuing your quest on renaming all of the test methods. I wish I had the time to figure out how to make telescope ignore methods in certain modules to stop you in your tracks.

Reply:
Please be my guest :innocent: